src/index.js

```diff
@@ -1,4 +1,4 @@
-export function h(type, props) {
+export function h(tag, props) {
```

Comment:
With argument name changes like `tag` and `from`/`source` will we merge this first and have others update their current PRs? Just wondering!

Reply:
@selfup Good question. I want to get some feedback here first, but what you say sounds good to me. It's just a tiny change anyway.